app/models/hmpps_api/offender_base.rb

```diff
@@ -255,15 +255,8 @@ def criminal_sentence?
       @sentence_type.civil? == false
     end
 
-    # Take either the new LocalDeliveryUnit (if available and enabled) and
-    # fall back to the old local_divisional_unit if not. This should all go away
-    # in Feb 2021 after the PDU changes have been rolled out in nDelius
     def ldu
-      if @case_information&.local_delivery_unit&.enabled?
-        @case_information.local_delivery_unit
-      else
-        @case_information&.local_divisional_unit
-      end
+      @case_information&.ldu
```

Comment:
This entire `Offender#ldu` method might be a bit pointless now. Could we lose this and simply use `@case_information&.ldu` in the `ldu_email` and `ldu_name` methods?